### DISCUSSION TITLE
CMake get_solution failed for the latest gcc version

### DISCUSSION THREAD

**Discussion by zee-doubleu-ee** [ANLAB-KAIST/KENSv3#237] [Project Q&A 4. Etc] [answered]
![kens-build-fail](https://github.com/ANLAB-KAIST/KENSv3/assets/73657186/03ad15ef-4053-4915-840d-290c85adbb0b)
Maybe this is because I'm using latest version of gcc. Is it ok to use the solution from gcc 13?

**Comment by Pusnow** [comment, marked as answer]
> Is it ok to use the solution from gcc 13?

Yes. PA4 does not use the solution binary, so it would be ok.
